docs/usage.md

```diff
@@ -25,6 +25,7 @@ The following query parameters can be used to send a *reconfigure* request to *D
 |reqMode        |The request mode. The proxy should be able to work with any mode supported by HAProxy. However, actively supported and tested modes are `http`, `tcp`, and `sni`. The `sni` mode implies TCP with an SNI-based routing.<br>Example: `tcp`|No|http|
 |reqPathReplace |A regular expression to apply the modification. If specified, `reqPathSearch` needs to be set as well.<br>Example: `/demo/`|No| |
 |reqPathSearch  |A regular expression to search the content to be replaced. If specified, `reqPathReplace` needs to be set as well.<br>Example: `/something/`|No| |
+|ServiceDefaultBackend  | If true, the service will be set to the default_backend rule, meaning it will catch all requests not matching any other rules.<br>Example: `true`|No| |
```

Comment:
`ServiceDefaultBackend` might be confusing. People might expect to put some value other than `true`. Maybe it would be better to name it `isDefaultBackend`?